### PR TITLE
Avatar,badge

### PR DESCRIPTION
**Pull request by ShoaibTamboli**
Avatar and Badge created. Will modify overall CSS later.

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    margin: 0 10px;
+}  
+
+.sm{
+    width: 40px;
+    height: 40px;
+}
+
+.md{
+    width: 50px;
```
Comment (Bibin-Antony) [inline components/Avatar/avatar.css:14]: you must replace all px with rem

File: style.css [hunk 2]
```diff
@@ -1,11 +1,139 @@
...
+    ;
+    --secondary-color: #39393b;
+    --primary-hover: #6f42c1;
+    --secondary-hover: #6c757d;
+    --white: #fff;
+    --grey: #6c757d;
+}
+
+main {
+    max-width: 700px;
```
Comment (Bibin-Antony) [inline style.css:24]: here too.

**Review by Bibin-Antony — APPROVED** [review]
kindly make the changes said.

File: components/Badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    position: absolute;
+    height: 12px;
+    width: 12px;
+    border-radius: 50%;
+    border: 2px solid white;
+    background-color: red
+}
+
+.badge .online{
+    background-color: goldenrod;
```
Comment (sumitch29) [inline components/Badge/badge.css:19]: Give proper color to online it should be green and also use color variables.

**Review by sumitch29 — APPROVED** [review]

**Pull request merged**